tests/container_tests/test_zip_container.py

```diff
@@ -485,6 +486,28 @@ def test_stat_non_exist(self, path_or_prefix):
             with self.assertRaises(FileNotFoundError):
                 handler.stat(path_or_prefix)
 
+    def test_stat_file(self):
+        test_file_name = 'testdir2/testfile1'
+        with self.fs_handler.open_as_container(self.zip_file_path) as handler:
+            stat = handler.stat(test_file_name)
+            self.assertIsInstance(stat, pfio.containers.zip.ZipFileStat)
+            self.assertTrue(stat.filename.endswith(test_file_name))
+            self.assertEqual(stat.size, 22)
```

Comment:
How about we retrieve the data from the `zip_info` so that these tests are more robust to changes?